com.sublimetext.three.yaml

```diff
@@ -83,8 +83,8 @@ modules:
       - type: extra-data
         filename: Package Control.sublime-package
         url: http://packagecontrol.io/Package%20Control.sublime-package
-        sha256: 6f4c264a24d933ce70df5dedcf1dcaeeebe013ee18cced0ef93d5f746d80ef60
-        size: 286331
+        sha256: 4e0764963c9abc0fe3c1f8fbb70f9c8cf3a14cf3f9ef62bf5175233b02008499
```

Comment:
This is a different size and checksum than I get when I download the file myself: I have `817937144c34c84c88cd43b85318b2656f9c3fac02f8f72cbc18360b2c26d139` and 471460. Could it be that they have changed the file again in the intervening time?

Reply:
> Could it be that they have changed the file again in the intervening time?

Fortunately I did not delete the file when I downloaded it to get size and checksum and yes, I can confirm that the one I download now is a different file from the one I downloaded previously. Seems they changed it again. I'll push the new shasum and size in a moment.